tools/sentieon_LongReadSV.cwl

```diff
@@ -0,0 +1,134 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: sentieon_LongReadSV
+doc: |-
+  Sentieon SV calling for PacBio HiFi and Oxford Nanopore long reads.
+  
+  ### Inputs:
+  #### Required
+  - ``Reference``: Location of the reference FASTA file.
+  - ``Input BAM``: Location of the BAM/CRAM input file.
+  - ``Platform``: PacBio HiFi or Oxford Nanopore
+
+requirements:
+- class: ShellCommandRequirement
+- class: InlineJavascriptRequirement
+- class: ResourceRequirement
+  coresMin: $(inputs.cpu) 
+  ramMin: $(inputs.ram * 1000) 
+- class: DockerRequirement
+  dockerPull: pgc-images.sbgenomics.com/hdchen/sentieon:202112.06
+- class: EnvVarRequirement
+  envDef:
+  - envName: SENTIEON_LICENSE
+    envValue: $(inputs.sentieon_license)
+baseCommand:
+- sentieon
+- driver
+arguments:
+- prefix: '--algo'
+  position: 10
+  valueFrom: LongReadSV
+  shellQuote: false
+inputs:
+  sentieon_license:
+    type: 'string'
+    doc: License server host and port
+  reference:
+    type: 'File'
+    secondaryFiles:
+    - pattern: .fai
+      required: true
+    - pattern: ^.dict
+      required: true
+    inputBinding:
+      prefix: -r
+      position: 0
+      shellQuote: false
+    doc: Reference fasta with associated fai index
+    sbg:fileTypes: FA, FASTA
+  input_bam:
+    type: 'File'
+    secondaryFiles:
+    - pattern: ^.bai
+      required: false
+    - pattern: ^.crai
+      required: false
+    - pattern: .bai
+      required: false
+    - pattern: .crai
+      required: false
+    inputBinding:
+      prefix: -i
+      position: 1
+      shellQuote: false
```

Comment:
Guessing this was written by senteion? otherwise might suggest `input_align` over `input_bam`

Reply:
Yup haodong wrote this one.